roles/compute_build_nodes_cloud/tasks/main.yml

```diff
@@ -0,0 +1,95 @@
+---
+
+#   - name: print single node info
+#     debug:
+#       var: item.mac
+#     with_items: "{{ compute_nodes }}"
+#   - name: print single node info
+#     debug:
+#       var: item.vnfs
+#     with_items: "{{ compute_nodes }}"
+#
+#   - fail:
+#       msg: "Quick fail for test!"
+
+   - block:
+     
+     - name: Check if node is already in warewulf database
+       command: sudo wwsh node list
+       register: wwsh_node_list
+       ignore_errors: yes
+
+     - name: Add node(s) into warewulf database
+       command: wwsh -y node new {{ item.name }} --ipaddr={{ item.ip }} --hwaddr={{ item.mac }} -D {{ compute_private_nic }}
+       when: "'{{ item.name }}' not in wwsh_node_list.stdout"
+       with_items: "{{ compute_nodes }}"
+
+     - name: Register node IP and MAC address in warewulf database
+       command: sudo wwsh node set {{ item.name }} -y -I {{ item.ip }} --hwadd {{ item.mac }}
+       when: "'{{ item.name }}' in wwsh_node_list.stdout"
+       with_items: "{{ compute_nodes }}"
+
+     - name: set nodes bootloader
+       command: wwsh -y object modify -s bootloader=sda -t node {{ item.name }}
+       with_items: "{{ compute_nodes }}"
+       when: stateful_nodes == true
+  
+     - name: set nodes partitions
+       command: wwsh -y object modify -s diskpartition=sda -t node {{ item.name }}
+       with_items: "{{ compute_nodes }}"
+       when: stateful_nodes == true
+  
+     - name: format partitions
+       command: wwsh -y object modify -s diskformat=sda1,sda2,sda3 -t node {{ item.name }}
+       with_items: "{{ compute_nodes }}"
+       when: stateful_nodes == true
+  
+     - name: define filesystems
+       command: wwsh -y object modify -s filesystems="{{ sda1 }},{{ sda2 }},{{ sda3 }}" -t node {{ item.name }}
+       with_items: "{{ compute_nodes }}"
+       when: stateful_nodes == true
+  #" for vim
+
+     - name: set files to provision
+       command: wwsh -y provision set {{ item.name }} --vnfs={{ item.vnfs }} --bootstrap={{ build_kernel_ver }} --files=passwd,group,shadow,munge.key,slurm.conf,dynamic_hosts,network --bootlocal=exit
```

Comment:
Since we are not using vnfs image in `_cloud` base build process, could you remove this?

Reply:
```
fatal: [ohpc]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'vnfs'\n\nThe error appears to be in '/CRI_XCBC/roles/compute_build_nodes_cloud/tasks/main.yml': line 53, column 8, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n     - name: set files to provision\n       ^ here\n"}
```
Of course, we could passing empty string to that variable by `vnfs: ""`, but we are not using it and also it's a brand new role, i think we should just remove it from this role.